_data/meltano/extractors/tap-pxwebapi/storebrand.yml

```diff
@@ -0,0 +1,25 @@
+capabilities: []
+description: PxWebApi - API for official statistics used by Statistical offices in
+  several Nordic countries
+domain_url: https://www.ssb.no/en/api/pxwebapi
+executable: tap-pxwebapi
+keywords: []
+label: Official statistics through PxWebApi
+logo_url: /assets/logos/extractors/pxwebapi.png
+maintenance_status: active
+name: tap-pxwebapi
+namespace: tap_pxwebapi
+next_steps: ''
+pip_url: git+https://github.com/storebrand/tap-pxwebapi.git
+quality: silver
+repo: https://github.com/storebrand/tap-pxwebapi
+settings:
+- kind: string
```

Comment:
@hholgersen would you mind adding label and description to these settings? Those are what populate the hub page. Label is usually just a more human readable version of `name` like base_url -> Base URL